### DISCUSSION TITLE
Generating Data in Azure Data Studio: inconsistencies between the generated date of birth (DOB) and the calculated age

### DISCUSSION THREAD

**Discussion by ifelseJAY** [microsoft/azuredatastudio#25495] [Q&A]
I am generating sample data in Azure Data Studio and have inconsistencies between the generated date of birth (DOB) and the calculated age. Despite attempts to correct the script, the age does not correspond accurately with the DOB. This mismatch suggests that the method used to generate DOB and calculate age might not be synchronized properly. how can I ensure that the DOB generation aligns with the age calculation to produce accurate results?

**Comment by erinstellato-ms** [comment]
@ifelseJAY - Can you please provide a sample script of what you're doing, including the table definition?  This shouldn't be specific to ADS, this would T-SQL and data type specific.
